problems/numberOfDigits.js

```diff
@@ -7,8 +7,14 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits(n) {
+    let count = 0 
+    let iter = n.toString()
+    
+    for (let num of iter){
+      count += 1
```

Comment:
What property does a string have that will provide you with the number of characters in the string? This property will remove the need for the for loop here.